resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala

```diff
@@ -381,4 +381,42 @@ object KubernetesUtils extends Logging {
       }
     }
   }
+
+  /**
+   * This function builds the EnvVar objects for each key-value env.
+   */
+  @Since("3.4.0")
+  def buildEnvVars(env: Map[String, String]): Seq[EnvVar] = {
+    if (env.isEmpty) {
```

Comment:
I'm wondering why we need this. This doesn't look like `Scalatic` if `else` is enough to handle this case.

Reply:
`else` was removed in https://github.com/apache/spark/pull/35886/commits/cfb25897c87d7fef0ead07067b7194d60989851f